src/adapters/networkhandler/mockNetworkHandler.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable import/no-dynamic-require */
+/* eslint-disable global-require */
+const path = require('path');
+const { getIntegrations } = require("../../routes/utils");
+// const { SUPPORTED_VERSIONS } = require("../../routes/utils/constants");
+
+const mockResponses = {};
+
+// Dynamically import the network handlers for all
+// the supported destinations
+const destinations = getIntegrations(path.resolve(__dirname, `../mocks`));
+
+destinations.forEach((dest) => {
+  mockResponses[dest] = require(`../mocks/${dest}/response.json`);
+});
+
+
+const sleep = async (sleepInMs) => new Promise((resolve) => { setTimeout(resolve, sleepInMs) });
+
+function mockNetworkHandler (type, originalNetworkHandler) {
```

Comment:
Can we have an example of how this will be used?

Reply:
I have added documentation for the same purpose. Can you once check and let me know ?

Reply:
Do you know if this is meant for testing by running the transformer service? is there the use of writing unit test cases using this?

Reply:
The intention with which we started out was by running the transformer service